custom/conf/app.ini.sample

```diff
@@ -383,6 +383,8 @@ MIN_PASSWORD_LENGTH = 6
 IMPORT_LOCAL_PATHS = false
 ; Set to true to prevent all users (including admin) from creating custom git hooks
 DISABLE_GIT_HOOKS = false
+; Set to false to allow pushes to gitea repositories despite having an incomplete environment - DISCOURAGED
```

Comment:
I would suggest the phrasing `NOT RECOMMENDED` in place of `DISCOURAGED`
```suggestion
; Set to false to allow pushes to gitea repositories despite having an incomplete environment - NOT RECOMMENDED
```

Reply:
Fair enough.